sdkproject/Assets/Mapbox/Examples/Scripts/TileErrorHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
```

Comment:
let's put this in the examples namespace and rename the class to make it obvious this is an example of how to register for these events.

Reply:
How does HandleTileErrorsExample sound as a class name?

Reply:
Sounds great!